datacosmos/const.py

```diff
@@ -1,5 +1,88 @@
 """Stores constants used in the datacosmos package."""
 
+from enum import Enum
+
 DATACOSMOS_PRODUCTION_BASE_URL = "https://app.open-cosmos.com/api/data/v0"
 DATACOSMOS_PRODUCTION_AUDIENCE = "https://beeapp.open-cosmos.com"
 DATACOSMOS_TOKEN_URL = "https://opencosmos.eu.auth0.com/oauth/token"
+
+
+class Satellites(Enum):
```

Comment:
Will these read like something like `from datacosmos.const import Satellites` `Satellites.SENTINEL_1A`? I would have preferred something like `from datacosmos.satellite import SENTINEL_1A` and it would avoid creating a poorly-scoped `const` file which is going to be become very large and be a dumping ground for many things.

Reply:
Yeah we can do it that way, python Enums are not great so getting rid of them is no bad thing, it's just a shame to lose the informative type hint

Reply:
That's a fair point. Is there a way we can eat our cake and have it too, i.e. avoid the enum but keep nice type hints? Python equivalent of `type Satellite string` in Go?